docs/faq.md

```diff
@@ -142,3 +142,32 @@ first download the file to the SCT folder, and execute the following command:
 ```bash
 ./docker/env/hydra.sh 'bash -c "sudo pip install pystack; pystack core core.python3.1000.bd43fbcd0c4b44488ce7e97e25fe1a28.1804.1745768005000000"'
 ```
+
+
+## How to define parameters for performance throughput test ?
```

Comment:
we should have separate docs for perf tests.
Can be introduced in followup.

Reply:
@juliayakovlev lets raise this in qa-task, there are multiple case of people complaining about not enough docs for how to use/write performance test

Reply:
https://github.com/scylladb/scylla-cluster-tests/issues/11216